cmake/Platform/Windows/PackagingCodeSign_windows.cmake

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+message(STATUS "Executing package signing...")
```

Comment:
same here, this message should be inside the function, otherwise it will print when the file is included